components/profile/LoginForm.tsx

```diff
@@ -20,6 +21,34 @@ const LoginForm = () => {
     []
   );
 
+  let logging_in;
+  if (typeof window !== "undefined"){
+    const code = new URLSearchParams(window.location.search).get("code");
+    if (code){
+      logging_in = (<p>Redirecting to home page...</p>);
+      React.useEffect(() => {
```

Comment:
Import the useState function from react instead of `React.useState`. Makes the code more easy to read.